utils/db.ts

```diff
@@ -286,11 +279,29 @@ export async function deleteVote(vote: Vote) {
     vote.user.id,
   ];
 
+  const [votedItemsByUserRes, votedUsersByItemRes] = await kv.getMany([
```

Comment:
this is something i stumbled upon by accident.  
if we compare `createVote` and `deleteVote`, we were missing a check for the `votedItemsByUserKey` and `votedUsersByItemKey` results.

as the deletion for a non-existent key doesn't throw (anymore?), the mutation on the `vote.item.score` would've been abused via api calls.

this is why an explicit check for these keys and their respective results has been introduced.

Reply:
Good catch! Though, next time, let's keep the objectives per PR to 1 🙂